subscription-contracts/src/MoonDAOTeamCreator.sol

```diff
@@ -64,54 +119,74 @@ contract MoonDAOTeamCreator is Ownable {
         openAccess = _openAccess;
     }
 
-    function createMoonDAOTeam(string memory adminHatURI, string memory managerHatURI, string memory memberHatURI, string calldata name, string calldata bio, string calldata image, string calldata twitter, string calldata communications, string calldata website, string calldata _view, string memory formId, address[] memory members) external payable returns (uint256 tokenId, uint256 childHatId) {
+    function createMoonDAOTeamFor(
+        address creator,
+        HatURIs memory hatURIs,
+        TeamMetadata calldata metadata,
+        address[] memory members
+    ) external payable onlyAuthorizedSigner returns (uint256 tokenId, uint256 childHatId) {
+        return _createMoonDAOTeam(creator, hatURIs, metadata, members);
+    }
 
-        require(whitelist.isWhitelisted(msg.sender) || openAccess, "Only whitelisted addresses can create a MoonDAO Team");
-        
+    function createMoonDAOTeam(
```

Comment:
bless you for simplifying the call parameters. this constructor was destroying my compiler 🙏